src/main/java/duke/UI.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+public class UI {
+    public static String breakLine = "____________________________________________________________\n";
+    public static String greeting = "Hello, I'm LishBot v6.9!\n" + "How may I help you today?\n";
+    public static String taskListOpening = "Finding your task list...\n" + "Found it! Here are what you have to do:\n";
+    public static String noListFound = "Congrats! You have finished all your task!\n";
+    public static String findingRelatedTask = "Let me find tasks that match your description...\n";
+    public static String noRelatedTaskFound = "I cannot find any tasks that match that description :(\n";
+    public static String goodBye = "Glad to be of help! See you later~\n";
+    public static String markAsDone = "Great! You have finished a task. I will mark this as done\n";
+    public static String markAsNotDone = "Okay. I will mark this task as not done yet\n";
+    public static String removeTask = "Roger that! I will remove this task\n";
+    public static String numberOfTaskLeft = "Now, the number of tasks you have is ";
+    public static String addTask = "Got it! I will add that task now\n";
+    public static String fileNotFound = "I cannot find the data file \n";
+    public static String createFile = "But don't worry I will create one for you :)\n";
```

Comment:
Do consider that some of these can have the final flag and have their names fit the coding standard for constants.
```suggestion
    public static final String MSG_LINE_BREAK = "____________________________________________________________\n";
    public static final String MSG_GREETING = "Hello, I'm LishBot v6.9!\n" + "How may I help you today?\n";
    public static final String MSG_TASK_LIST = "Finding your task list...\n" + "Found it! Here are what you have to do:\n";
    public static final String MSG_CONGRATS = "Congrats! You have finished all your task!\n";
    public static final String MSG_FIND_MATCH = "Let me find tasks that match your description...\n";
    public static final String MSG_GOODBYE = "Glad to be of help! See you later~\n";
    public static final String MSG_MARK_DONE = "Great! You have finished a task. I will mark this as done\n";
    public static final String MSG_MARK_NOT_DONE = "Okay. I will mark this task as not done yet\n";
    public static final String MSG_REMOVE_TASK = "Roger that! I will remove this task\n";
    public static final String MSG_NUMBER_OF_TASKS = "Now, the number of tasks you have is ";
    public static final String MSG_ADD_TASK = "Got it! I will add that task now\n";
    public static final String MSG_FILE_CREATED = "But don't worry I will create one for you :)\n";
        
    public static final String ERR_CANNOT_FIND = "I cannot find any tasks that match that description :(\n";
    public static final String ERR_DATA_NOT_FOUND = "I cannot find the data file \n";
```